service/routes.py

```diff
@@ -48,3 +48,54 @@ def index():
 ######################################################################
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
+
+
+######################################################################
+# CREATE A NEW WISHLIST
+######################################################################
+@app.route("/wishlists", methods=["POST"])
+def create_wishlists():
+    """
+    Creates a Wishlist
+    This endpoint will create a Wishlist based the data in the body that is posted
+    """
+    app.logger.info("Request to create a Wishlist")
+    check_content_type("application/json")
```

Comment:
Do not use that function. It's incorrect.

To check if body is JSON: https://github.com/CSCI-GA-2820-FA25-001/wishlists/pull/16#discussion_r2412040506